code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -391,6 +397,36 @@ var/global/list/damage_icon_parts = list()
 
 	if(update_icons)   update_icons()
 
+//VOREstation edit
+/mob/living/carbon/human/proc/update_belly(var/update_icons=1)

```

Comment:
It can. The important part is that the belly code MUST be allocated its own layer because previous attempts to add the image onto other layers have always failed, and I'm not certain they can succeed as images and icons can't be blended together, but the back layer always returns an image.